content/language/golang/configure-ci-cd.md

```diff
@@ -25,11 +25,11 @@ Create a GitHub repository, configure the Docker Hub secrets, and push your sour
 2. Open the repository **Settings**, and go to **Secrets and variables** >
    **Actions**.
 
-3. Create a new secret named `DOCKER_USERNAME` and your Docker ID as value.
+3. Create a new **Repository secrets** named `DOCKER_USERNAME` and your Docker ID as value.
 
-4. Create a new [Personal Access Token (PAT)](../../security/for-developers/access-tokens.md/#create-an-access-token) for Docker Hub. You can name this token `docker-tutorial`.
+4. Create a new [Personal Access Token (PAT)](../../security/for-developers/access-tokens.md/#create-an-access-token) for Docker Hub. You can name this token `docker-tutorial`. Make sure access permissions include Read, Write and Delete.
```

Comment:
Don't think it needs `Delete` perm

Reply:
Yes, you are correct. I have updated it. Thank you for the review.